src/lib/diff.js

```diff
@@ -213,6 +223,7 @@ function diffSources(before, after, commands, sourcesRemoved, differ) {
       } else {
         // no update command, must remove then add
         updateSource(sourceId, after, commands, sourcesRemoved);
+        // TODO Update differ?
```

Comment:
Would you mind expanding on this? It isn't obvious to me what's actionable here / should there be an issue for it / would it be completely superseded by #19.